icarusalg/gallery/MCTruthBase/MCTruthAssociations.cpp

```diff
@@ -560,11 +560,12 @@ double MCTruthAssociations::length(const detinfo::DetectorPropertiesData& detPro
 {
     // Get fiducial volume boundary.
     double xmin = -0.1;
-    double xmax = 2.*fGeometry->DetHalfWidth() + 0.1;
```

Comment:
Are these calls the same????

Reply:
Yes, the calls are the same: `fGeometry->DetHalfWidth()` (etc.) was forwarding directly to `fGeometry->TPC({0, 0}).HalfWidth()`.  Now it's just explicit.